change/@ni-nimble-components-5e4aad3f-b6d2-47fa-8d61-7c6f4415f6b9.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
While you're at it, could you also update `table-column.stories.ts` to ensure that both the dedicated grouping and sorting stories have similar language? Looks like the sorting one already says "Sorting is based on the underlying field values in the column, not the rendered values" but `groupedRowsDescription` doesn't mention it.